packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.component.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Action } from '../../Actions';
+import theme from './InputTitleSubHeader.scss';
+import Icon from '../../Icon';
+
+function DetailsTitle({ title, subTitle, onEdit }) {
+	return (
+		<span className={classNames(theme['tc-subheader-details-text'], 'tc-subheader-details-text')}>
+			<span className={theme['subheader-details-title-icon-container']}>
```

Comment:
Be more concise with your class like `tc-subheader-icon-container`

And please double classNames with plain text

```jsx
<span className={classNames(theme['tc-subheader-icon-container'], 'tc-subheader-icon-container')}>
```